crates/nu-command/tests/commands/math/round.rs

```diff
@@ -14,8 +14,38 @@ fn can_round_very_large_numbers() {
 fn can_round_very_large_numbers_with_precision() {
     let actual = nu!(
         cwd: ".",
-        "echo 18.13725447800741422899276654867720121457878988 | math round -p 10"
+        "18.13725447800741422899276654867720121457878988 | math round -p 10"
     );
 
     assert_eq!(actual.out, "18.137254478")
 }
+
+#[test]
+fn can_round_integer_with_negative_precision() {
+    let actual = nu!(
+        cwd: ".",
+        "123 | math round -p -1"
+    );
+
+    assert_eq!(actual.out, "120")
+}
+
+#[test]
+fn can_round_float_with_negative_precision() {
+    let actual = nu!(
+        cwd: ".",
+        "123.3 | math round -p -1"
+    );
+
+    assert_eq!(actual.out, "120")
+}
```

Comment:
Maybe worth moving one of the negative precision tests directly to the examples so the user can see it as a possibility. (also the example test are a bit faster to run in CI and locally)